### ISSUE TITLE
Merge Dockerfiles

### ISSUE THREAD

**Issue by HexF** [akaunting/docker#4]
To accompany #2 it would be better to fit both NGINX and PHP into the same single Dockerfile.
From quickly glancing over the Dockerfiles, I can't see a reason why it wouldn't work in fact it would work better.

The nginx Dockerfile refers to the [php container by name](https://github.com/akaunting/docker/blob/90874012411a2044c048f66442e4f0424acbcc9a/nginx/conf.d/akaunting.conf#L42) which would cause problems split over 2 Dockerfiles as a user might not install the other one, or could call it something else.

Currently the way it works is with `docker-compose`, which builds the images and starts them. This means that the image could not be put on Docker Hub / quay.io as it would depend on another container to have an exact name, which is not always possible, one such instance would be when running it on a Kubernetes Cluster.

TL;DR - Merge docker files and remove explicit references between multiple images

**Comment by hmert** [comment]
@HexF thank you for your commend. 

we will merge php and nginx into one Docker image and also we'll have soon all-in-one image that includes DB.

**Comment by bd8392** [comment]
I would suggest to let the database live outside... because in most cases, the accounting software is linked to a crm or a website, that already have a database...an "all in one docker" should not be the way to go for the official container...it also would be harder to maintain...but maybe you can provide a working docker-compose with mariadb :-) A good idea is to link php and nginx/Apache inside (as most distros on docker-hub do) @carlos-jenkins would that fit your needs? :-)

**Comment by josephhernandezphd** [comment]
I recommend launching the db internally and putting it on a service network. I’m my opinion, It’s easier to maintain and link the crm back to the akaounting db. 

I also recommend merging the two files. Both containers use the same mount point and both have read / write access creating a potential conflict/file corruption.

**Comment by bd8392** [comment]
@HexF would this help? https://github.com/akaunting/docker/issues/2#issuecomment-649593005

**Comment by denisdulici** [comment]
We're looking for maintainers and would appreciate your help #13
